Client/Ecosia/UI/NTP/AboutEcosia/NTPAboutEcosiaCell.swift

```diff
@@ -0,0 +1,224 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+final class NTPAboutEcosiaCell: UICollectionViewCell, ReusableCell {
+    struct UX {
+        static let height: CGFloat = 64
```

Comment:
You're right, the height is actually accessed from outside and that is why I created it, but if I did I should move the other things inside, will do 🙂

Reply:
I started doing but this view has so many values (and I actually reused the one existing at the `YourImpact` page). I will then leave it as is with only height in the UX since it is accessed from outside if you don't mind... When we discuss styling and wether to keep them we can refactor if needed.